package.json

```diff
@@ -37,9 +37,11 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "phantomjs-prebuilt": "^2.1.7",
-    "react": "^15.0.0",
```

Comment:
That depends on what you're asking about - devDependencies are only for when you're working *on* the code, issuing `npm install` specifically in its own dir, not for when you're using the code as a dependency in a larger project. When npm installs something as a normal library installation, the devDependencies don't resolve at all, so you should be able to use `npm link` in your personal React clone's dir and as a normal library react-onclickoutside should now be using your React, not the one indicated in the package.json.

Only if you do dev work on this code, will it use the "regular" React because the package.json instructs it to explicitly install it as full project install.

So I'd still like to know the exact use case *you* ran into (what were you trying to do/how were you using the library) that necessitates moving the dependency to a peerDependency, which makes dev work considerably more annoying because it will force people who work on this code to always manually install `react` and `react-dom` in addition to running `npm install` (peer dependencies are *not* installed by npm anymore since v3)